### ISSUE TITLE
Outdated Documentation (How to use this library?)

### ISSUE THREAD

**Issue by polaski** [AFNetworking/AFAmazonS3Manager#42]
I don't want to come off as the guy who just takes it for granted, but just to let you know I've been trying to get this to work for the last several hours and still couldn't. I'm not an idiot and i've never had this severe issue when integrating 3rd party libraries before.

What I've figured out so far is that I think the example code in Readme is inconsistent with how the code is actually implemented--even the filenames are different.

I'm guessing there used to be one AFAmazonS3Client but now it's separated into AFAmazonS3Manager and AFAmazonS3RequestSerializer. Even after figuring that out I am having hard time actually using the library.

Basically I have no idea what values I should be passing for what parameters. I've done my homework and searched around the web for examples of how people use it but they're all outdated. Could someone please explain how to use this?
